modules/spo-creating-profiles.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-creating-profiles_{context}"]
+= Creating {type} profiles
+
+Use the `{kind}` object to create profiles.
+
+ifdef::seccomp[]
+.Procedure
+
+* The {type} profile will be saved in `/var/lib/kubelet/{type}/operator/<namespace>/<name>.json`.
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1beta1
+kind: {kind}
+metadata:
+  namespace: example-namespace
+  name: example-profile
+spec:
+  defaultAction: SCMP_ACT_LOG
+----
+
+An `init` container creates the root directory of the Security Profiles Operator to run the Operator without `root` group or user ID privileges. A symbolic link is created from the rootless profile storage `/var/lib/openshift-security-profiles` to the default `seccomp` root path inside of the kubelet root `/var/lib/kubelet/{type}/operator`.
+endif::[]
+ifdef::selinux[]
+The `{kind}` object has several features that allow for better security hardening and readability. The `{kind}` object:
+
+* Restricts the profiles to inherit from to the current namespace or a system-wide profile. Because there are typically many profiles installed on the system, but only a subset should be used by cluster workloads, the inheritable system profiles are listed in the `spod` instance in `spec.selinuxOptions.allowedSystemProfiles`.
+* Performs basic validation of the permissions, classes and labels.
+* Adds a new keyword `@self` that describes the process using the policy. This allows reusing a policy between workloads and namespaces easily, as the usage of the policy is based on the name and namespace.
+
+.Procedure
+
+. Create a policy that can be used with a non-privileged workload by creating the following `{kind}` object:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha2
+kind: {kind}
+metadata:
+  name: nginx-secure
+  namespace: nginx-deploy
+spec:
+  allow:
+    '@self':
+      tcp_socket:
+      - listen
+    http_cache_port_t:
+      tcp_socket:
+      - name_bind
+    node_t:
+      tcp_socket:
+      - node_bind
+  inherit:
+  - kind: System
+    name: container
+----
+
+. Wait for `selinuxd` to install the policy by running the following command:
++
+[source,terminal]
+----
+$ oc wait --for=condition=ready selinuxprofile nginx-secure
```

Comment:
Let's add the namespace to wait on explicitly so that the examples can be copy-pasted:
```
oc wait --for=condition=ready -nnginx-deploy selinuxprofile nginx-secure
```